app/helpers/role_helper.rb

```diff
@@ -0,0 +1,10 @@
+module RoleHelper
+  def user_role(role)
+    users = User.all
+    user = role.users.ids
+    user&.each do |id|
+      users = users.where.not(id: id)
+    end
+    users
+  end
+end
```

Comment:
I'm confused about what this method does. Should it be named `users_without_role` instead?

Would this do the same thing?
```
def users_without_role(role)
  User.where.not(id: role.users.ids)
end
```